pegen/python_generator.py

```diff
@@ -107,6 +108,13 @@ def visit_Repeat1(self, node: Repeat1) -> Tuple[str, str]:
         self.cache[node] = name, f"self.{name}()"  # But no trailing comma here!
         return self.cache[node]
 
+    def visit_RepeatWithSeparator(self, node: RepeatWithSeparator) -> Tuple[str, str]:
+        if node in self.cache:
+            return self.cache[node]
+        name = self.gen.name_node_with_sep(node)
+        self.cache[node] = name, f"self.{name}()"  # But no trailing comma here!
```

Comment:
I'd change this comment to "# No trailing comma here either!" (so the sequence of comments in these 4 methods reads somewhat humorously).